sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/FileLines20.cs

```diff
@@ -6,22 +6,21 @@ public class FooBar
         public FooBar()
         {
             System
-                   .
-                   Collections
-
+                   .Collections
```

Comment:
Why are you changing the old behavior?

Reply:
Duncan raised a good point in the other PR: https://github.com/SonarSource/sonar-csharp/pull/1100#discussion_r163596548

This is to make this rule consistent with the other.

The thinking is that a function with 5 strings spanning 10 lines each should be considered to have 50 lines, not 10.